src/pages/components/select.mdx

```diff
@@ -0,0 +1,101 @@
+---
+title: Select
+description: >
+  Select allows a user to pick one option from a list.
+  This helps simplify the UI when space is limited.
+package: '@zendeskgarden/react-dropdowns'
+propSheets:
+  - dropdowns/src/elements/Select/Select.tsx
+---
+
+<Usage>
+  <Use>
+    <ul>
+      <li>To make a selection from a list of options</li>
+    </ul>
+  </Use>
+  <Misuse>
+    <ul>
+      <li>
+        To filter a large list of options, consider an{' '}
+        <a href="/components/autocomplete">Autocomplete</a>
+      </li>
+      <li>
+        To make multiple selection from a list, use a{' '}
```

Comment:
Should be "To make multiple selections" (selection needs to be made plural)

Reply:
Copy has been updated, thanks!